docs/tool/js/views/map-view.js

```diff
@@ -869,6 +871,59 @@
                 .remove();
 
         },
+        exportCsv: function() {
+          // if (data){
+            d3.select('#csvExport')
+              .on('click', function(d) {
+                  console.log("YOU CLICKED EXPORT CSV");
+                  var preview = d3.select('#buildings-list');
+                  var listItems = preview.selectAll('div');
+                  var projectNames = [];
+                  listItems._groups[0].forEach( function(project){
+                    projectNames.push(project.innerText);
+                  })
+                  var allData = mapView.convertedProjects.features;
+                  // console.log(mapView.convertedProjects);
+                  var matchesData = allData.filter(function(feature) {
+                      return feature.properties.matches_filters === true;
+                  });
+                  var notMatchesData = allData.filter(function(feature) {
+                      return feature.properties.matches_filters === false;
+                  });
+
+                  //Create a csv from the data
+                  var csvContent = "data:text/csv;charset=utf-8,";
+                  var keys = "matches_filters,";
+                  for (key in allData[0].properties){
+                    if ( key !== "matches_filters"){
+                      keys += key + ",";
+                    }
+                  }
+                  csvContent += keys + '\n';
+                  matchesData.forEach( function(project){
+                    csvContent += String(project.properties['matches_filters']) + ",";
+                    for (key in project.properties){
+                      if ( key !== "matches_filters"){
+                        csvContent += String(project.properties[key]).replace(/,/g,' ') + ",";
```

Comment:
@jgordo04 I'd vote for going ahead and adding papaparse as a dependency to make this quick, easy, and robust. Maybe put a comment in the code beside the html js include statement that says this is just used for creating csv export format and could be removed if that was done manually, in case we ever want to hunt to lighten our footprint in the future.

Reply:
Done